dashboard/app/helpers/levels_helper.rb

```diff
@@ -25,7 +25,7 @@ def build_script_level_path(script_level, params = {})
     elsif script_level.stage.lockable?
       script_lockable_stage_script_level_path(script_level.script, script_level.stage, script_level, params)
     elsif script_level.bonus
-      query_params = params.merge(id: script_level.id)
+      query_params = params.merge(level_name: script_level.level.name)
```

Comment:
What will this affect? Is there a test you can write to show what was wrong with the old behavior?

Reply:
levels are indexed by name, so that's good.

 level name "should" be unique according to rails validations and other assumptions in our system, however this is not enforced at the database level. I think you can probably proceed with this and we can take making that index unique as a follow-up item.

Reply:
https://codedotorg.atlassian.net/browse/PLAT-10

Reply:
> Is there a test you can write to show what was wrong with the old behavior?

The old behavior wasn't "wrong" per se, it just didn't work well between environments. In this case, we want to be able to generate urls from the i18n machine that will still work on prod, which wouldn't work under the old system because ids are not consistent while names are.

I'll add a test

> https://codedotorg.atlassian.net/browse/PLAT-10

Thanks!